components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -58,61 +58,57 @@ export function StartWorkspaceModal(props: StartWorkspaceModalProps) {
         return true;
     }, [repo, selectedIde, selectedWsClass, useLatestIde, errorIde, errorWsClass]);
 
-    const buttons = useMemo(() => {
-        const result = [
-            <button key="cancel" className="secondary" onClick={props.onClose}>
-                Cancel
-            </button>,
-            <button
-                key="start"
-                className=""
-                onClick={startWorkspace}
-                disabled={!repo || repo.length === 0 || !!errorIde || !!errorWsClass}
-            >
-                New Workspace
-            </button>,
-        ];
-        if (!props.onClose) {
-            return result.slice(1, 2);
-        }
-        return result;
-    }, [props.onClose, startWorkspace, repo, errorIde, errorWsClass]);
-
     return (
         <Modal
             onClose={props.onClose || (() => {})}
             closeable={!!props.onClose}
             onEnter={startWorkspace}
             visible={true}
-            title="Open in Gitpod"
-            buttons={buttons}
         >
-            <div className="-mx-6 px-6">
-                <div className="text-xs text-gray-500">Start a new workspace with the following options.</div>
-                <div className="pt-3">
-                    <RepositoryFinder
-                        setSelection={props.contextUrl && !props.allowContextUrlChange ? undefined : setRepo}
-                        initialValue={repo}
-                    />
-                </div>
-                <div className="pt-3">
-                    {errorIde && <div className="text-red-500 text-sm">{errorIde}</div>}
-                    <SelectIDEComponent
-                        onSelectionChange={onSelectEditorChange}
-                        setError={setErrorIde}
-                        selectedIdeOption={selectedIde}
-                        useLatest={useLatestIde}
-                    />
-                </div>
-                <div className="pt-3">
-                    {errorWsClass && <div className="text-red-500 text-sm">{errorWsClass}</div>}
-                    <SelectWorkspaceClassComponent
-                        onSelectionChange={setSelectedWsClass}
-                        setError={setErrorWsClass}
-                        selectedWorkspaceClass={selectedWsClass}
-                    />
+            <ModalHeader>Open in Gitpod</ModalHeader>
+            <ModalBody noScroll>
```

Comment:
updated to use the composition components, and added a noScroll flag to keep some of the behavior that this modal needs due to the dropdown components.